axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+| `axon.tracing.attributeProviders.aggregateIdentifier` | `true`  | Whether to add the aggregate identifier as a label when handling a message                             |
+| `axon.tracing.attributeProviders.messageId`           | `true`  | Whether to add the message identifier as a label when handling a message                               |
+| `axon.tracing.attributeProviders.messageName`         | `true`  | Whether to add the message name as a label when handling a message                                     |
+| `axon.tracing.attributeProviders.messageType`         | `true`  | Whether to add the message type as a label when handling a message                                     |
+| `axon.tracing.attributeProviders.payloadType`         | `true`  | Whether to add the payload type as a label when handling a message                                     |
+| `axon.tracing.attributeProviders.metadata`            | `true`  | Whether to add the metadata properties as labels when handling a message                               |
+
+#### Manual configuration
+
+The OpenTelemetry support can also be configured using the `Configurer` of Axon Framework to configure
+the `OpenTelemetrySpanFactory`.
+
+```java
+public class AxonConfigurer {
+
+    public void configure(Configurer configurer) {
+        configurer.defaultConfiguration()
+                  .configureSpanFactory(c -> OpenTelemetrySpanFactory.builder().build());
+    }
+}
+```
+
+Note that when not using Spring boot, tracing each message handler invocation is not supported due to a limitation.
+
+## OpenTracing <a id="opentracing"></a>
+
+OpenTracing is deprecated. If necessary, you can
+use [the OpenTracing extension to Axon Framework](../../extensions/tracing.md).
+The functionality of the extension is rather limited compared to the OpenTelemetry integration, so using that if
+possible is preferred.
+
+## Logging <a id="logging"></a>
+
+Sometimes you don't have an APM system available, for instance during local development. It might still be useful to see
+the traces that would be started and finished to obtain insights. For this purpose, the framework provides
+a `LoggingSpanFactory`.
+
+You can configure the `LoggingSpanFactory` in the following ways:
+{% tabs %}
+{% tab title="Spring Boot Bean" %}
+
+```java
+@Configuration
+public class AxonConfiguration {
+
+    @Bean
+    public SpanFactory spanFactory() {
+        return LoggingSpanFactory.INSTANCE;
+    }
+}
+```
+
+{% endtab %}
+{% tab title="Manual" %}
+
+```java
+public class AxonConfigurer {
+
+    public void configure(Configuration configuration) {
+        configuration.configureSpanFactory(c -> LoggingSpanFactory.INSTANCE);
+    }
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Traced components
+
+Axon Framework provides a large range of components that are traced by the configured `SpanFactory`.
+Each component and their added spans are available for reference in this section, with additional information about how
```

Comment:
```suggestion
Each component, and their added spans, are available for reference in this section, with additional information about how
```

Reply:
Changed wording, but I don't like the two comma approach. Grammarly says both are valid